### ISSUE TITLE
[BUG]: accent marks are not displayed correctly

### ISSUE THREAD

**Issue by razielfayad** [martinrotter/rssguard#1292]
### Brief description of the issue

The accent marks are not displayed correctly in some posts. So far, it only seems to affect the Xataka feed. Checking the list of articles, it seems that the issue started in December.. The problem only occurs through Inoreader. If I directly add the feed, it works perfectly. Related issue #268.

### How to reproduce the bug?

1. open RSS Guard
2. Wait for the new articles to synchronize or check the articles from the last month

### What was the expected result?

The accent marks should be displayed without issues.

RSS Guard using classic feed

![Screenshot_ 5845 _RSS_Guard_4 6 3_2024-02-04_12-30-01](https://github.com/martinrotter/rssguard/assets/22458962/30485e43-7b78-4dc7-a3b3-5a0637af0d97)

Inoreader

![Screenshot_(622)_Inoreader_-_Todos_los_artículos_-_Brave_2024-02-04_12-34-14](https://github.com/martinrotter/rssguard/assets/22458962/ad88d9ef-a832-4b0f-b1da-c35298a9234c)

### What actually happened?

Different characters are being displayed.

RSS Guard using Inoreader feed

![Screenshot_ 5856 _RSS_Guard_4 6 3_2024-02-04_12-28-16](https://github.com/martinrotter/rssguard/assets/22458962/37126d22-2c72-40db-a49a-e2dfefdd09ee)

### Debug log

[rssguard.log](https://github.com/martinrotter/rssguard/files/14157966/rssguard.log)

### Operating system and version

* OS: Windows 10 22H2
* RSS Guard version: 4.6.3
* Revisión: 43d164dc4-lite

**Comment by martinrotter** [comment]
Well. Inoreader now requires "PRO" to use their API? If that is the case, I am afraid I am unable to test Inoreader integration properly, I do not have their PRO subscription.

**Comment by razielfayad** [comment]
Yes, it seems that since they implemented OAuth 2.0, they started charging for access to the API. I still have access to the API with a free Inoreader account, maybe because I created it before the change, but I'm afraid this could change at any time. Is there any way to switch Inoreader sources to local without losing unread articles?

**Comment by martinrotter** [comment]
It seems to be fixed for me, tested now with development [master] branch.
